R/snapshot.R

```diff
@@ -960,8 +958,9 @@ renv_snapshot_dependencies_impl <- function(project, type = NULL, dev = FALSE) {
         ""
       )
 
-      renv_scope_options(renv.verbose = TRUE)
-      writef(lines, renv_difftime_format(elapsed))
+      # force output in this scope
+      renv_scope_caution(TRUE)
```

Comment:
Isn't this the default?

Reply:
It is, but it's forced TRUE here because we want to display this message even if caution was disabled (e.g. as we do in the "synchronized" check)